locales/typed/pnpm-lock.yaml

```diff
@@ -0,0 +1,332 @@
+lockfileVersion: '9.0'
```

Comment:
These generated files, both the locale type file and these lockfiles, should be marked as generated in `.gitattributes`

e.g. https://github.com/scpwiki/wikijump/blob/develop/deepwell/.gitattributes

Reply:
Apparently `pnpm-lock.yaml` isn't defined in framerail gitattributes either, rip

Reply:
updated in: [6600895](https://github.com/scpwiki/wikijump/pull/2031/commits/66008951c1c5a265a8aacb22469beeadf01d26f4)

Reply:
No no, don't gitignore it, it should be in the repo, but in git**attributes** you should tell git "this is a binary file" or "this is a vendored/automatically generated file" so it can generate diffs appropriately

Reply:
I think i figured it out.